### PR TITLE
added advisor conditionals

### PR DESCRIPTION
**Pull request by stackpoint**
Thanks to Marshmellerz13 on discord: https://discordapp.com/channels/174949513000255488/695648572024029274/699058689604190258

### CODE REVIEW

File: common/advisortypes/Idea_Variation_advisors.txt [hunk 1]
```diff
@@ -452,6 +467,14 @@ idea_var_advisor_23 = {
 
 	chance = {
 		factor = 1
+		
+		modifier = {
+			factor = 0
+			owner = {
+				is_subject = yes 
+				NOT = { is_subject_of_type = tributary_state }
+			}
```
Comment (afowl) [inline common/advisortypes/Idea_Variation_advisors.txt]: subjects province war score cost do apply to your peace deal

**Comment by stackpoint** [comment]
I need to do some testing on whether current_institution_growth works the way I believe it does.

**Comment by afowl** [comment]
It breaks the game, somehow. If you reload the save it deletes all countries

File: common/advisortypes/Idea_Variation_advisors.txt (hunk 1 above)
Comment (stackpoint) [reply common/advisortypes/Idea_Variation_advisors.txt]: Thanks, I'll remove this restriction then.

**Comment by stackpoint** [comment]
current_institution_growth causes the save to corrupt

**Comment by stackpoint** [comment]
Just kidding, turns out I just misspelled modifier

**Comment by stackpoint** [comment]
Typo fixed and advisor scopes narrowed a bit more. If someone else can test, it can be pushed to live.

**Comment by stackpoint** [comment]
I think this is ready to go. Although we may want to consider lowering the advisor pool by 1 (to 5) because of how the pool is being narrowed.

**Pull request merged**